source/features/clean-pinned-issues.css

```diff
@@ -26,11 +35,13 @@
 		display: table-cell !important;
 		padding: 6px 12px;
 		vertical-align: middle;
+		white-space: nowrap;
```

Comment:
This avoids some awkward wrapping in certain cases

<img width="121" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/5f597bd8-1198-4615-a45d-d076e3d32f93">